contracts/instruments/RibbonVolatility.sol

```diff
@@ -459,12 +459,48 @@ contract RibbonVolatility is DSMath, InstrumentStorageV1, InstrumentStorageV2 {
         emit Exercised(msg.sender, positionID, totalProfit, optionsExercised);
     }
 
-    function claimRewards(string calldata adapterName, address rewardsAddress, uint256[] calldata optionIDs)
+    function claimRewards(string calldata adapterName, address rewardsAddress)
         external
     {
       IProtocolAdapter adapter = IProtocolAdapter(factory.getAdapter(adapterName));
-      adapter.delegateClaimRewards(rewardsAddress, optionIDs);
-      emit ClaimedRewards(optionIDs.length);
+      uint256[] memory optionIDs = getOptionIDs(msg.sender);
+      uint256 claimedRewards = adapter.delegateClaimRewards(rewardsAddress, optionIDs);
+      emit ClaimedRewards(claimedRewards);
+    }
+
+    function rewardsClaimable(string calldata adapterName, address rewardsAddress)
+        external
+        view
+        returns (uint256 rewardsToClaim)
+    {
+      IProtocolAdapter adapter = IProtocolAdapter(factory.getAdapter(adapterName));
+      uint256[] memory optionIDs = getOptionIDs(msg.sender);
+      rewardsToClaim = adapter.delegateRewardsClaimable(rewardsAddress, optionIDs);
```

Comment:
Refer to: https://github.com/ribbon-finance/structured-products/pull/59/files#r582650439

Can call the function directly:
```
rewardsToClaim = adapter.rewardsClaimable(rewardsAddress, optionIDs);
```

Reply:
yes, tried it initially but rewardsClaimable was not found as its not in IProtocolAdapter or ProtocolAdapter. We can either keep way it is now or add rewardsClaimable stub in IProtocolAdapter and add empty function in MockGammaAdapter / GammaAdapter.

Reply:
Okay gotcha, I see where you're coming from. Let's stick with this approach.

Reply:
I do want to find a way in the future to call specific adapter functions without completely bloating the IProtocolAdapter. Need more thought there. But let's go with this for now.